### DISCUSSION TITLE
app.UseClientSideBlazorFiles<Client.Startup>(); and Microsoft.AspNetCore.Blazor.Server

### DISCUSSION THREAD

**Discussion by ShaunCurtis** [dotnet/aspnetcore#26215] [General] [answered]
Is Microsoft.AspNetCore.Blazor.Server still live?  If not how do we do what UseClientSideBlazorFiles did for mapping multiple WASM SPAs on the same server.  There's a DanRoth Github repo at https://github.com/danroth27/BlazorWebAssemblyWithPrerendering that uses it.

**Comment by SteveSandersonMS** [comment, marked as answer]
The `Microsoft.AspNetCore.Blazor.Server` namespace was renamed to `Microsoft.AspNetCore.Components.Server` a long time ago, before .NET 3 shipped.

The replacement for `UseClientSideBlazorFiles` is `UseBlazorFrameworkFiles`.

**Comment by ShaunCurtis** [comment]
Thanks Steve, will go and take a look at the code.  Away, so apologies for not getting back sooner.  Closed.
